contracts/marketplace/Makefile.toml

```diff
@@ -53,16 +48,13 @@ script = '''
 [tasks.accounts]
 dependencies = ["create_account_admin", "create_traders"]
 
-[tasks.get_token]
+[tasks.create_asset]
 plugin = "retry_script"
 script = '''
-soroban lab token id --asset native --network ${network} > /tmp/token_id.txt
-'''
-
-[tasks.get_asset]
-plugin = "retry_script"
-script = '''
-soroban lab token id --asset native --network ${network} > /tmp/asset_id.txt
+  soroban config identity generate asset_issuer_account
+  curl "${friendbot}?addr=$(soroban config identity address asset_issuer_account)"
+  ./asset_test_gen/asset_test_gen -i $(soroban config identity show asset_issuer_account) -d $(soroban config identity show trader_1) -r $(soroban config identity show trader_2),$(soroban config identity show trader_3) > /tmp/asset_address_pair.txt
```

Comment:
I was following this [docs](https://developers.stellar.org/docs/issuing-assets/how-to-issue-an-asset#full-code-sample), and there is no Rust example there. I went for the Go one because its the only one compiled language there. Would love to have that done in Rust, but for the size/time available, Go was a direct win.